charts/lakefs/templates/service.yaml

```diff
@@ -9,6 +9,9 @@ spec:
   ports:
     - port: {{ .Values.service.port }}
       targetPort: http
+{{- if .Values.service.nodePort }}
```

Comment:
please keep the `if` statement at the same ident level as the field itself

Reply:
Updated indentation